Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BASE_IMAGE=alpine:3.14.3
+ARG BASE_IMAGE=alpine:3.15.4
```

Comment:
shall we just use alpine:3.15 ?

Reply:
so subminor updates come automatically

Reply:
I prefer to stick to exact  version to have the same image every build